bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/parser/Converter.java

```diff
@@ -9,14 +9,21 @@
 
 import java.util.Dictionary;
 
+/**
+ * This interfase provides opportunity to use different parsers for converting JsonObject to Dictionary.
+ *
+ * @author Vasil Ilchev - Initial Contribution
+ *
+ */
 public interface Converter {
```

Comment:
They are different things. The parser parses JSON to DTO objects, the Converter parses string value of context parameter of ConfigDescriptionParameter when resolve connection between custom and system module types